taui/package.json

```diff
@@ -14,7 +14,7 @@
     "prestart": "yarn",
     "pretest": "yarn",
     "semantic-release": "semantic-release",
-    "start": "mastarm build --serve --watch",
+    "start": "mastarm build --serve --watch --proxy http://django:8085",
```

Comment:
One thing I noticed is that if you try to access `/api/` through the proxy (i.e. `localhost:9966/api/`) the static files don't show up and return 404s. However, I don't think we need to fix that because we're planning to shift away from Mastarm anyway, and everything works fine if you access the Django container directly via `localhost:8085/api/`.